demisto_sdk/commands/validate/validators/BA_validators/BA111_is_entity_name_contain_excluded_word.py

```diff
@@ -0,0 +1,110 @@
+from demisto_sdk.commands.content_graph.objects.incident_field import IncidentField
+from demisto_sdk.commands.content_graph.objects.incident_type import IncidentType
+from demisto_sdk.commands.content_graph.objects.indicator_field import IndicatorField
+from demisto_sdk.commands.content_graph.objects.indicator_type import IndicatorType
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.job import Job
+from demisto_sdk.commands.content_graph.objects.layout import Layout
+from demisto_sdk.commands.content_graph.objects.layout_rule import LayoutRule
+from demisto_sdk.commands.content_graph.objects.mapper import Mapper
+from demisto_sdk.commands.content_graph.objects.modeling_rule import ModelingRule
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.objects.parsing_rule import ParsingRule
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.objects.report import Report
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.content_graph.objects.test_playbook import TestPlaybook
+from demisto_sdk.commands.content_graph.objects.trigger import Trigger
+from demisto_sdk.commands.content_graph.objects.widget import Widget
+from demisto_sdk.commands.content_graph.objects.wizard import Wizard
+from demisto_sdk.commands.content_graph.objects.xsiam_dashboard import XSIAMDashboard
+from demisto_sdk.commands.content_graph.objects.xsiam_report import XSIAMReport
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[
+    Integration,
+    Script,
+    Playbook,
+    Pack,
+    Dashboard,
+    Classifier,
+    Job,
+    Layout,
+    Mapper,
+    Wizard,
+    CorrelationRule,
+    IncidentField,
+    IncidentType,
+    IndicatorField,
+    IndicatorType,
+    LayoutRule,
+    Layout,
+    ModelingRule,
+    ParsingRule,
+    Report,
+    TestPlaybook,
+    Trigger,
+    Widget,
+    GenericDefinition,
+    GenericField,
+    GenericModule,
+    GenericType,
+    XSIAMDashboard,
+    XSIAMReport,
+]
+
+
+def name_does_contain_excluded_word(content_item: ContentTypes) -> bool:
+
+    lowercase_name = content_item.display_name.lower()
+
+    return any(
+        (
+            excluded_word in lowercase_name
+            for excluded_word in EXCLUDED_DISPLAY_NAME_WORDS
+        )
+    )
+
+
+class IsEntityNameContainExcludedWordValidator(BaseValidator[ContentTypes]):
+    error_code = "BA111"
+    description = "Checks whether given object contains excluded word."
+    error_message = "The content item name is {} , which contains an excluded word."
+    related_field = ""
+    is_auto_fixable = False
+    expected_git_statuses = [
+        GitStatuses.RENAMED,
+        GitStatuses.ADDED,
+        GitStatuses.MODIFIED,
+    ]
```

Comment:
if this is the default, I think we can remove it

Reply:
I don't see a default value to this attribute

Reply:
There's an implicit default in `base_validator.py`. Empty value means all statuses.

```python
def should_run_according_to_status(
    content_item_git_status: Optional[GitStatuses],
    expected_git_statuses: Optional[List[GitStatuses]],
) -> bool:
    return not expected_git_statuses or content_item_git_status in expected_git_statuses
```

Reply:
There is another status we don't need to validate ->  the deleted files,